app/scripts/views/mixins/settings-panel-mixin.js

```diff
@@ -53,4 +53,24 @@ define(function (require, exports, module) {
       self.closePanel();
     }
   };
+  if (window.history && window.history.pushState) {
+    $(window).on('popstate', function () {
+      var hashLocation = location.hash;
+      var hashSplit = hashLocation.split('#!/');
+      var hashName = hashSplit[1];
+
+      if (hashName !== '') {
+        var hash = window.location.hash;
+        if (hash === '') {
+          var openElements = $('.settings-child-view').find('.open');
+          if (openElements !== '' || openElements !== undefined){
```

Comment:
I think [jQuery's find](https://api.jquery.com/find/) will always return a jQuery element collection, so I'm unsure of whether this line is needed.